public/index.html

```diff
@@ -88,7 +58,29 @@ <h1 id="title">LaTeX Online</h1>
 <a href="https://github.com/aslushnikov/latex-online#ccommand-line-interface">Tell
 me more</a>
                         </li>
+                    </ol>                    
+                    <h3>Parameters</h3>
+                    <ol>
+                        <li><code>&command=</code> xelatex or pdflatex</li>
```

Comment:
I think it's important to explain about those parameters right on the main page as it explains well the functionality.
I added one sentence with explanation - if you still think it's unclear - feel free to delete the entire paragraph because there is already 4 links to documentation.